Sources/KSCrashHandler.swift

```diff
@@ -159,9 +85,95 @@ internal class KSCrashHandler: CrashHandler {
 			userInfo[keyBreadcrumbsSerialized] = breadcrumbsSerialized
 		}
 
-		installation?.userInfo = userInfo
+		KSCrash.sharedInstance().userInfo = userInfo
+	}
+
+}
+
+class KSCrashSentryInstallation: KSCrashInstallation {
+	
+	private let generatedEvent: GeneratedEvent
+	
+	init(generatedEvent: GeneratedEvent) {
+		self.generatedEvent = generatedEvent
+		super.init(requiredProperties: [])
 	}
+	
+	override func sink() -> KSCrashReportFilter! {
+		return KSCrashReportSinkSentry(generatedEvent: generatedEvent)
+	}
+	
+}
 
+class KSCrashReportSinkSentry: NSObject, KSCrashReportFilter {
+	
+	private let generatedEvent: GeneratedEvent
+	
+	init(generatedEvent: GeneratedEvent) {
+		self.generatedEvent = generatedEvent
+		super.init()
+	}
+	
+	func filterReports(reports: [AnyObject]!, onCompletion: KSCrashReportFilterCompletion!) {
+		
+		// Mapping reports
+		let events = reports?
+			.flatMap({$0 as? CrashDictionary})
+			.map({mapReportToEvent($0)})
+		
+		// Propigating this generated event up so the SentryClient object can send it off
+		for event in events ?? [] {
+			generatedEvent(event: event)
+		}
+		
+		onCompletion?(reports, true, nil)
```

Comment:
This feels a little awkward. There's no propagation of errors from attempting to send events to the server. Might be better to put the sending code inside the sink rather than relying on a callback.